lib/s3_meta_sync/syncer.rb

```diff
@@ -197,11 +205,19 @@ def delete_local_files(local, paths)
     end
 
     def s3
-      @s3 ||= Aws::S3::Client.new(
-        access_key_id: @config[:key],
-        secret_access_key: @config[:secret],
-        region: @config[:region] || 'us-west-2'
-      )
+      @s3 ||= begin
+        config = { region: @config[:region] || DEFAULT_REGION }
+
+        if @config[:credentials_path] && File.exist?(@config[:credentials_path])
```

Comment:
if the user set a path that does not exist, better blow up ?
```suggestion
        if @config[:credentials_path]
```